http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/HttpAdvancedIT.java

```diff
@@ -15,7 +15,8 @@
 public class HttpAdvancedIT extends BaseHttpAdvancedIT {
 
     @KeycloakContainer(command = { "start-dev", "--import-realm", "--hostname-strict=false" })
-    static KeycloakService keycloak = new KeycloakService(DEFAULT_REALM_FILE, DEFAULT_REALM, DEFAULT_REALM_BASE_PATH);
+    static KeycloakService keycloak = new KeycloakService(DEFAULT_REALM_FILE, DEFAULT_REALM, DEFAULT_REALM_BASE_PATH)
+            .withProperty("JAVA_OPTS", "-Xms512m -Xmx1g");
```

Comment:
1. Yes it's the only test affected. The other ones are passing for instance `KeycloakWebappSecurityIT` or `HttpAdvancedReactiveIT`... I am currently investigating this by isolating different tests one by one in `BaseHttpAdvancedIT` to figure out which one is consuming the memory resources in the Keycloak container.

2. Ok restoring to the initial base code.

Reply:
> Yes it's the only test affected. 

That is strange. Any idea why?

Reply:
It seems this test is causing the issue:
https://github.com/quarkus-qe/quarkus-test-suite/blob/6de7fb98a148e02db062f79ff1e41238496df0b3/http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/BaseHttpAdvancedIT.java#L149

Reply:
It's weird, because I commented that test and Jenkins job passed, but according to the annotation that test is just executed for  `@EnabledOnQuarkusVersion(version = "1\\..*"`...and it's not the case...

Reply:
Right, it's some remnant that we forget and needs to be dropped, but it is highly unlikely it causes anything. Maybe Jenkins simply succeeded because it's flaky test.

Reply:
As I mentioned above that's the test root cause issue, I've recently tested again with @Disabled that test and PASSED in Jenkins job, so I will remove the test as @michalvavrik suggests.

Reply:
dropping this code makes sense, but I recommend to test that this fixes the issue on long series of Jenkins run

Reply:
> So, why is this the only test, which is affected?

+1

Reply:
OK, this takes forever. @jcarranzan please:

1.  Create an issue in the TS to investigate why this tests (and only it ) fails
2. Add link to the issue to a TODO in this PR
And let's finally merge it.

Reply:
ok, also I realized that fails in Podman Jenkins job machines, I will describe my insights in the issue.

Reply:
Do not forget to add the link to a todo comment here